PhotoPlusFrontend/PhotoPlus/src/app/models/address/countryCode.ts

```diff
@@ -0,0 +1,511 @@
+        codes.add(this.BM);
+        codes.add(this.BN);
+        codes.add(this.BO);
+        codes.add(this.RO);
+        codes.add(this.BQ);
+        codes.add(this.BR);
+        codes.add(this.BS);
+        codes.add(this.BT);
+        codes.add(this.RS);
+        codes.add(this.BV);
+        codes.add(this.BW);
+        codes.add(this.RU);
+        codes.add(this.BY);
+        codes.add(this.BZ);
+        codes.add(this.RW);
+        codes.add(this.SA);
+        codes.add(this.SB);
+        codes.add(this.CA);
+        codes.add(this.SC);
+        codes.add(this.CC);
+        codes.add(this.CD);
+        codes.add(this.SD);
+        codes.add(this.CF);
+        codes.add(this.CG);
+        codes.add(this.CH);
+        codes.add(this.CI);
+        codes.add(this.SE);
+        codes.add(this.CK);
+        codes.add(this.CL);
+        codes.add(this.CM);
+        codes.add(this.CN);
+        codes.add(this.CO);
+        codes.add(this.SG);
+        codes.add(this.SH);
+        codes.add(this.CR);
+        codes.add(this.SI);
+        codes.add(this.SJ);
+        codes.add(this.CU);
+        codes.add(this.CV);
+        codes.add(this.CW);
+        codes.add(this.CX);
+        codes.add(this.CY);
+        codes.add(this.CZ);
+        codes.add(this.SK);
+        codes.add(this.SL);
+        codes.add(this.SM);
+        codes.add(this.SN);
+        codes.add(this.SO);
+        codes.add(this.SR);
+        codes.add(this.SS);
+        codes.add(this.ST);
+        codes.add(this.SV);
+        codes.add(this.DE);
+        codes.add(this.SX);
+        codes.add(this.SY);
+        codes.add(this.SZ);
+        codes.add(this.TC);
+        codes.add(this.DJ);
+        codes.add(this.DK);
+        codes.add(this.TD);
+        codes.add(this.DM);
+        codes.add(this.TF);
+        codes.add(this.DO);
+        codes.add(this.TG);
+        codes.add(this.TH);
+        codes.add(this.TJ);
+        codes.add(this.TK);
+        codes.add(this.TL);
+        codes.add(this.TM);
+        codes.add(this.TN);
+        codes.add(this.TO);
+        codes.add(this.TR);
+        codes.add(this.TT);
+        codes.add(this.DZ);
+        codes.add(this.TV);
+        codes.add(this.TW);
+        codes.add(this.TZ);
+        codes.add(this.UA);
+        codes.add(this.EC);
+        codes.add(this.EE);
+        codes.add(this.UG);
+        codes.add(this.EG);
+        codes.add(this.EH);
+        codes.add(this.UM);
+        codes.add(this.US);
+        codes.add(this.ER);
+        codes.add(this.ES);
+        codes.add(this.ET);
+        codes.add(this.UY);
+        codes.add(this.UZ);
+        codes.add(this.VA);
+        codes.add(this.VC);
+        codes.add(this.VE);
+        codes.add(this.VG);
+        codes.add(this.VI);
+        codes.add(this.FI);
+        codes.add(this.FJ);
+        codes.add(this.FK);
+        codes.add(this.VN);
+        codes.add(this.FM);
+        codes.add(this.FO);
+        codes.add(this.FR);
+        codes.add(this.VU);
```

Comment:
pls

Reply:
Then how in other way You would see enum which provides country name and send country code to API?

Reply:
I've changed it a bit. Now the set is filled only once and object is shared so it should not be a problem. If You have any other ideas how to solve this feel free to share it.

Reply:
I was thinking that would be provided by backend, 
and be configurable depending on where shop would like to offer it services.

Reply:
It is possible but it would required saving of that information on backend permanently so probably in DB and allowing customisation via admin panel to allow choosing of countries where shipment is possible, so I took easier way to assume that we allow shipping globally and created class not enum, because with enum in Angular as far as I know You cannot read key value as string to be able to provide country name for frontend and code for backend. 
I tried:
```
enum Code{
PL = "Poland",
DE = "Germany"
}
and I did not found solution for reading enum value name as string.